test/inspect.test.js

```diff
@@ -157,6 +157,49 @@ test('inspect', function (t) {
     });
 });
 
+test('inspect setup.py', function (t) {
+  return Promise.resolve().then(function () {
+    chdirWorkspaces('setup_py-app');
+    t.teardown(testUtils.activateVirtualenv('pip-app'));
+  })
+    .then(function () {
+      return plugin.inspect('.', 'setup.py');
+    })
+    .then(function (result) {
+      var plugin = result.plugin;
+      var pkg = result.package;
+
+      t.test('plugin', function (t) {
+        t.ok(plugin, 'plugin');
+        t.equal(plugin.name, 'snyk-python-plugin', 'name');
+        t.match(plugin.runtime, 'Python', 'runtime');
+        t.notOk(plugin.targetFile, 'no targetfile for setup.py');
+        t.end();
+      });
+
+      t.test('package', function (t) {
+        t.ok(pkg, 'package');
+        t.equal(pkg.name, 'test_package', 'name');
+        t.equal(pkg.version, '1.0.2', 'version');
+        t.end();
+      });
+
+      t.test('package dependencies', function (t) {
+        Object.keys(pipAppExpectedDependencies).forEach(function (depName) {
+          t.match(
+            pkg.dependencies[depName],
+            pipAppExpectedDependencies[depName].data,
```

Comment:
should this still be `pipAppExpectedDependencies`? I would assume something like `setup_py-appExpectedDependencies`.

Reply:
Yes, now that there is the `provenance` thing, they won't be the same, indeed. I'll change it